docs/src/apps/jsonrpc-api.md

```diff
@@ -1016,6 +1020,118 @@ curl -X POST -H "Content-Type: application/json" -d '{"jsonrpc":"2.0", "id":1, "
 {"jsonrpc":"2.0","result":{"context":{"slot":1114},"value":{"circulating":16000,"nonCirculating":1000000,"nonCirculatingAccounts":["FEy8pTbP5fEoqMV1GdTz83byuA8EKByqYat1PKDgVAq5","9huDUZfxoJ7wGMTffUE7vh1xePqef7gyrLJu9NApncqA","3mi1GmwEE3zo2jmfDuzvjSX9ovRXsDUKHvsntpkhuLJ9","BYxEJTDerkaRWBem3XgnVcdhppktBXa2HbkHPKj2Ui4Z],total:1016000}},"id":1}
 ```
 
+### getTokenAccountBalance
+
+Returns the token balance of an SPL Token account.
+
+#### Parameters:
+
+- `<string>` - Pubkey of Token account to query, as base-58 encoded string
+- `<string>` - Pubkey of Token program ID that owns the account, as base-58 encoded string
+- `<object>` - (optional) [Commitment](jsonrpc-api.md#configuring-state-commitment)
+
+#### Results:
+
+- `RpcResponse<u64>` - RpcResponse JSON object with `value` field set to the balance
+
+#### Example:
+
+```bash
+// Request
+curl -X POST -H "Content-Type: application/json" -d '{"jsonrpc":"2.0", "id":1, "method":"getTokenAccountBalance", "params": ["7fUAJdStEuGbc3sM84cKRL6yYaaSstyLSU4ve5oovLS7", "TokenSVp5gheXUvJ6jGWGeCsgPKgnE3YgdGKRVCMY9o"]}' http://localhost:8899
+// Result
+{"jsonrpc":"2.0","result":{"context":{"slot":1114},"value":9864,"id":1}
+```
+
+### getTokenAccountsByDelegate
+
+Returns all SPL Token accounts by approved Delegate.
+
+#### Parameters:
+
+- `<string>` - Pubkey of account delegate to query, as base-58 encoded string
+- `<string>` - Pubkey of Token program ID that owns the accounts, as base-58 encoded string
+- `<object>` - (optional) Configuration object containing the following optional fields:
+  * (optional) [Commitment](jsonrpc-api.md#configuring-state-commitment)
+  * (optional) `mint: <string>` - specific token Mint to limit accounts to
+
+#### Results:
+
+The result will be an RpcResponse JSON object with `value` equal to an array of JSON objects, which will contain:
+
+- `pubkey: <string>` - the account Pubkey as base-58 encoded string
```

Comment:
I think we need to return the mint address as well for each account.  Otherwise the user needs to immediately look at up when they aren't searching under a specific mint

Reply:
Ah, actually the mint address is part of the account state, and I am returning parsedJson by default (ref: the example). Does that work?

Reply:
Ah, cool.